docs/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ function(add_nvqpp_test TEST_NAME SOURCE_LOCATION)
   NAME
     nvqpp_${TEST_NAME}
   COMMAND
-    bash -c "${CMAKE_BINARY_DIR}/bin/nvq++ ${NVQPP_COMPILE_ARGS} ${PARSED_ARGS_SOURCE_DIR}/${SOURCE_LOCATION} -o ${TEST_NAME} ;\
+    bash -c "${CMAKE_BINARY_DIR}/bin/nvq++ ${NVQPP_COMPILE_ARGS} ${PARSED_ARGS_SOURCE_DIR}/${SOURCE_LOCATION} -o ${TEST_NAME} && \
```

Comment:
Might want to `rm -f ${TEST_NAME}` just before this as well.

Reply:
I think the `&&` should make any additional cleanup (like `rm`) unnecessary. I did multiple tests with it to make sure it fails when compilation fails. (Plus, the `rm` command would emit an error if the file wasn't found, which could be confusing to people looking at the output when running `ctest ... -V`.

Reply:
The `-f` won't fail or issue an error.

the point of deleting the file would be to prevent the false positive scenario of the compiler being a NOP==`true` and running an old, stale file. Far-fetched perhaps, but we're talking about computers here. :)

Reply:
OK, updated in c9c58b17

Reply:
FWIW, many of our llvm-lit tests in tests/NVQPP would be equally vulnerable to these "far-fetched" scenarios, too.